server.js

```diff
@@ -15,8 +15,23 @@ var checkVersion = function(major, minor, patch) {
     return true;
 }
 
+
+// Only print if debug mode is on
+var log = {
+    debug: function (message) {
+        if (debug) {
+            console.info(message);
+            fs.write('phantomjs.log', message + '\n', 'a');
+        };
+    },
+    error: function (message) {
+        console.warn(message);
```

Comment:
`warn` is used instead of `error` because it `error` outputs line each time (in our case, it is always line 28 so not really useful) while `warn` only outputs the message itself